packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx

```diff
@@ -195,64 +207,75 @@ export const Account = ({
   const secondaryEmails = emails!.filter((email) => !email.isPrimary);
 
   return (
-    <section className="account" data-testid="account-section">
+    <section className="mt-8" data-testid="account-section">
       <ul>
-        <li className="flex justify-space-between">
-          <h3 data-testid="email-label">
+        <li className={`${styleClasses.li} flex justify-between`}>
+          <h3 data-testid="email-label" className="mt-0 my-0 mb-1 text-lg">
             <span
-              className={`${query === primaryEmail.email ? 'highlight' : ''}`}
+              className={`${
+                query === primaryEmail.email ? 'bg-yellow-100' : ''
+              }`}
```

Comment:
Another small nit: if we set an empty class name to an empty string like this in React, the "class" attribute is rendered on the DOM element with an empty string. If you set it to `undefined` instead, it'll just render `<span>` without the attribute.

Also, similarly to the other comment, I believe this can be simplified to:

`className={query === primaryEmail.email ? 'bg-yellow-100' : undefined}`

You could also consider using [classnames](https://www.npmjs.com/package/classnames) which we have in other packages but eh 🤷‍♀️